gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v3_16_0/schema.yml

```diff
@@ -11,3 +11,36 @@ databaseChangeLog:
                   type: nvarchar(16)
                   constraints:
                     nullable: true
+        - createTable:
+            tableName: ${gravitee_prefix}key_subscription
```

Comment:
About the table name, should it be key_subscription**s** ?

To match the other 'link' table, which are mostly at plural form.